src/components/geometry.js

```diff
@@ -143,6 +144,33 @@ function getGeometry (data, schema) {
     case 'plane': {
       return new THREE.PlaneBufferGeometry(data.width, data.height);
     }
+    case 'polygon': {
+      var verts = data.vertices.split(' ');
```

Comment:
we should have a `vec3Array` property type that handles this out-of-the-box where it looks like ```1 2 3, 2 4 5, 1 6 2```